osmaxx/profile/views.py

```diff
@@ -18,14 +18,15 @@
 class SendVerificationEmailMixin(object):
     RATE_LIMIT_SECONDS = 30
 
-    def _send_email_verification(self, profile):
-        if cache.get(profile.associated_user.id):
+    def _send_email_verification(self):
```

Comment:
I like that one much better, i puts readable names and clearer structure on the Profile module, which is the one that should be concerned with this, IMO.
I accidentally already merged it, if you don't agree with the change merge, I'll revert it - otherwise, I think we can close this issue.